outbox/event-routing-smt/src/main/java/io/debezium/examples/outbox/routingsmt/EventRouter.java

```diff
@@ -1,88 +0,0 @@
-/*
```

Comment:
Nice, nothing beats deleting code. Great sign that things are getting easier right out-of-the-box!

Reply:
Feels good right?! :smile: